lib/ecto/postgres/pg_rational_migrations.ex

```diff
@@ -204,16 +205,13 @@ defpgmodule Vtc.Ecto.Postgres.PgRational.Migrations do
       args: [input: :rational],
       returns: :rational,
       declares: [
-        greatest_denom: {:bigint, "gcd(input.numerator, input.denominator)"},
+        greatest_denom: {:bigint, "GCD(input.numerator, input.denominator)"},
         denominator: {:bigint, "ABS(input.denominator / greatest_denom)"},
         numerator: {:bigint, "input.numerator / greatest_denom"}
       ],
       body: """
-      IF (input).denominator < 0 THEN
-        RETURN  (numerator * -1, denominator);
-      ELSE
-        RETURN (numerator, denominator);
-      END IF;
+      numerator := numerator * sign((input).denominator);
```

Comment:
Noticed "sign" capitalized in other places, so modifying for consistency.
```suggestion
      numerator := numerator * SIGN((input).denominator);
```

Reply:
In f3fad1c2eef9b59b1988907567960b32690002eb